docs/userman/gui/pages/builder.md

```diff
@@ -126,6 +130,29 @@ Compared to the previous example, you can see that the label uses a bold font we
 
     The Page Builder API does support index values that are not Python identifiers or literals.
 
+!!! note "Lambdas as property values"
+    Using a lambda function as a property value simplifies the creation of dynamic user interfaces
+    where elements change based on run-time information. This approach is particularly useful when
+    you need to build loops or handle varying data.
+
+    For example, consider the following dictionary:
+    ```python
+    ages = {
+        "Albert": 58,
+        "Beatrix": 71,
+        "Cecilia": 23
+    }
+    ```
+    We want to generate a series of text elements displaying the name and age of each person in the
+    list. A lambda function makes this straightforward:<br/>
+    ```python
+    for name in ages:
+        tgb.text(lambda ages: f"{name} is {ages.get(name)}")
```

Comment:
This example is a bit misleading
Because we use two versions of the ages variable: 
The frame variable in the for
The state variable in the parameter of the lambda
So even if an entry is added to the state variable, the result will not change
But it will change if you change one value if the dict 

I would propose
prefix = "prefix: "
for name, age in ages.entries():
  tgb.text(lambda prefix: f"{prefix}{name} is {age}"

And show that you can change the prefix state variable

Reply:
That is SO true.
What about
```
import taipy.gui.builder as tgb
from taipy.gui import Gui

people = {
    "Albert": 1982,
    "Beatrix": 1955,
    "Cecilia": 2003
}
         
current_year = 2024

with tgb.Page() as page:
    tgb.text("Year: ", inline=True)
    tgb.slider("{current_year}", min=1990, max=2050)

    for name, birth_year in people.items():
        tgb.text(lambda current_year: f"{name} would be {current_year-birth_year}")

if __name__ == "__main__":
    Gui(page).run()
```

Reply:
Excellent